src/globals.c

```diff
@@ -35,6 +38,59 @@ char *elf_symtab;
 char *elf_strtab;
 char *elf_section;
 
+
+
+/*
+ * Enhence the find_func functions through trie_t struct.
+ *
+ * insert_trie_t:
+ * Parameters:
```

Comment:
Check https://www.kernel.org/doc/html/v4.10/doc-guide/kernel-doc.html for writing [Sphinx](http://www.sphinx-doc.org/) friendly comments.